test/TritonGPU/amd/optimize-lds-usage.mlir

```diff
@@ -44,29 +44,30 @@ module attributes {"ttg.num-warps" = 8 : i32, "ttg.threads-per-warp" = 64 : i32}
   }
 }
 
-// -----
+// // -----
```

Comment:
@antiagainst This PR is breaking these two lit tests. I think it's coming from the changes in `Allocation.cpp`, but I have not track them down. I assume the fix would be something like using `LinearEncodingAttr::getOrder` instead of `getOrder` somewhere or so.

Reply:
Feel free to land; we can fix later. @binarman FYI